docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,219 @@
+.. currentmodule:: discord
+
+.. _guide_topic_permissions:
+
+
+Permissions
+============
+
+Permissions are the bread and butter of how control over a discord server (which henceforth be referenced as "guild" for the sake of API terminology).
+
+Controlling permissions is necessary for securing and controlling how your guild functions and operates.
+
+
+Permission hierarchy explanation
+---------------------------------
+
+Discord permissions work on a hierarchy system. This means that roles placed logically above each other (in the Roles UI) have descending order of precedence.
+
+If you have a role list, like so:
+
+- ``Administrator (#1)``
+- ``Manager (#2)``
+- ``Moderator (#3)``
+- ``@everyone (#4)``
+
+This means that Administrator role (someone who has it) can kick, ban and timeout Manager.
+Manager **cannot** kick, ban or timeout Administrator.
+
+discord.py is aware of this and has implemented the :meth:`~Role.__gt__` and similar methods on :class:`~Role` that allows you to perform the following:
+
+.. code-block:: python3
+
+    if administrator_role > manager_role:
+        # The administrator role is greater in the hierarchy than manager.
+
+One small caveat to note with the discord permissions system, is that in the case of overwrites, explicit allow will always overwrite explicit deny.
+
+What I mean by this, is the following:
```

Comment:
Generally speaking I think by standard we want to always either address by mass or avoid self-referencing entirely.
```suggestion
What we mean by this, is the following:
```
```suggestion
What is meant by this, is the following:
```
```suggestion
In clearer terms, imagine this:
```